### ISSUE TITLE
Face Plate Visible When In First Person With First Person Mod

### ISSUE THREAD

**Issue by SkeleBones** [FiguraMC/Figura#195]
Basically, the face model I have on my avatar is visible when using the first person mod and in first person. This didn't use to be the case in older builds. 

![image](https://github.com/FiguraMC/Figura/assets/38815328/bb65cb32-1dc6-4347-b3d8-bbb943eba21e)

**Comment by SkeleBones** [comment]
I have attached the avatar file in case it is helpful. 
[Skin_v3.zip](https://github.com/FiguraMC/Figura/files/14470847/Skin_v3.zip)

**Comment by PoolloverNathan** [comment]
rename HEAD to Head

(edit: appears to be rotating with camera; seems like a real bug)

**Comment by SkeleBones** [comment]
> rename HEAD to Head
> 
> (edit: appears to be rotating with camera; seems like a real bug)

Yeah, when I look straight ahead and up it is fine, but when looking anywhere else it appears. Also, when changing HEAD to Head, it causes the script to not work.

**Comment by Gakuto1112** [comment]
I asked this problem to First-person Model developers.

https://github.com/tr7zw/FirstPersonModel/issues/431

In this conversation, the develop said that this problem may be related to Fabric or Figura because he didn't confirm this issue in Forge version. And, also said that Figura Fabric version may do something different from Forge versions.

### Environments

| Item | Version |
| - | - |
| Minecraft | 1.20.4 |
| Fabric Loader | 0.15.7 |
| Fabric API | 0.96.4+1.20.4 |
| First-person Model | 2.3.4 |
| Figura | 0.1.4+1.20.4 |

**Comment by Gakuto1112** [comment]
As a solution for now, you can downgrade First-person Model to 2.3.3.
